### PR TITLE
Change main "Getting Started" to "Quick Start"

### PR DESCRIPTION
**Pull request by snowkeeper**
As mentioned in #15 we have two Getting Started sections.  This switches the main section to Quick Start.

quickstart

footer

both

### CODE REVIEW

File: content/en/pages/home.jade [hunk 1]
```diff
@@ -5,8 +5,8 @@ block homepage-header
 		img(src='/images/logo-inverted.svg', alt='KeystoneJS', width=80, height=81)
 		h1 Node.js CMS &amp; Web Application Platform
 		h2 The open source framework for developing database-driven websites, applications and APIs in Node.js. Built on Express and MongoDB.
-		.home-header-buttons
-			a(href='/getting-started', title='KeystoneJS getting started guide').btn.btn-lg.btn-primary
+		.home-header-buttons 
+			a(href='/quick-start', title='KeystoneJS quick start guide').btn.btn-lg.btn-primary
```
Comment (JedWatson) [inline content/en/pages/home.jade:9]: doesn't this need to be mirrors in the `zh` version as well?

Comment (snowkeeper) [inline content/en/pages/home.jade:9]: It does. I can't change the actual text to match so left it for the time being. 

The links would all still function for search engine saneness.

Comment (JedWatson) [inline content/en/pages/home.jade:9]: We can update `href='/getting-started'` -> `href='/quick-start'` in the `zh` docs so the nav continues to work (just matching on the path string)... ?

Comment (snowkeeper) [inline content/en/pages/home.jade:9]: Ok, if you are happy with the look I will make those corrections and attach.

Comment (JedWatson) [inline content/en/pages/home.jade:9]: :+1:

Once this is merged maybe @weekwood can help with the rest of the translation

Comment (weekwood) [inline content/en/pages/home.jade:9]: sure 👍

发自我的 iPhone

**Pull request closed without merging**